net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild

```diff
@@ -22,6 +22,7 @@ fi
 
 RDEPEND="
 	dev-libs/glib:2
+	dev-util/glib-utils
```

Comment:
glib-utils is no `RDEPEND` for sure.

Reply:
Ok, thank you for feedback. I'll move it to `DEPEND`.